en/components/date_time_editor.md

```diff
@@ -0,0 +1,173 @@
+
+#### Binding 
+A basic configuration scenario setting a Date object as a [`value`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#value):
+```typescript
+public date = new Date();
+```
+
+```html
+<igx-input-group>
+    <input type="text" igxInput igxDateTimeEditor [value]="date"/>
+</igx-input-group>
+```
+
+To create a two-way data-binding, set an ngModel:
+```html
+<igx-input-group>
+    <input type="text" igxInput igxDateTimeEditor [(ngModel)]="date"/>
+</igx-input-group>
+```
+<div class="divider--half"></div>
+
+
+#### Keyboard Navigation
+Date Time Editor Directive has intuitive keyboard navigation that makes it easy to jump trough different DateParts, increment, decrement, etc. without having to touch the mouse.
+
+- <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>Arrow Left</kbd> / <kbd>Right</kbd> - navigates between date sections. Jumps from current section to the same caret position of the next/previous section. On start/end sections, first jump to the other end of the same section following the direction of the arrow.
+
+- <kbd>Arrow Up</kbd> / <kbd>Down</kbd> - increment/decrement date portions. See related [`isSpinLoop`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#isspinloop)
+
+- <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>;</kbd> - set the current day and time in the editor via a shortcut (as it is in Excel).
+
+
+### Examples
+
+#### Display and input format
+Date Time Editor Directive support different display and input formats. Notice formats like shortDate, longDate, etc., can be used as displayFormat only:
+
+```html
+<igx-input-group>
+    <input type="text" igxInput [displayFormat]="'shortDate'" [igxDateTimeEditor]="'dd/MM/yyyy'" [(ngModel)]="date"/>
+</igx-input-group>
+```
+
+#### Min max value
+What is more, you can specify [`minValue`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#minvalue) and [`maxValue`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html#maxvalue) properties to restrict input and control the validity of the ngModel.
+```typescript
+public minDate = new Date(2020, 1, 15);
+public maxDate = new Date(2020, 12, 1);
+``` 
+
+```html
+<igx-input-group>
+    <input type="text" igxInput igxDateTimeEditor [minValue]="minDate" [maxValue]="maxDate" [(ngModel)]="date" required/>
+</igx-input-group>
+
+```
+
+#### Increment and decrement
+`igxDateTimeEditor` directive exposes public `increment` and `decrement` methods. These increment/decrement a specific `DatePart` of the currently set date and time and can be used in a couple of ways.
+
+In the first scenario, if no specific DatePart is passed to the method, a default DatePart will increment/decrement, based on the specified `inputFormat` and the internal directive implementation.
```

Comment:
Here we could also say that `igxDateTimeEditor` will prioritize `date` over anything else if it is present in the input format, if `hours` is present in the input format, it will be prioritized after `date`.
E.G. 
12/10/2015 13:45:06 -> increment() -> 13/10/2015 13:45:06
13:45:06 -> increment() -> 14:45:06

Reply:
I will omit this description as it is both an edge case(use spin btns without passing DatePart) as well as there is a tiny chance of changing this behavior in the future.